### PR TITLE
Replaced all /en Links 

### LINKED ISSUE #3475: Replace all /en links
Find all /en links in the repo and remove the locale part of the link

### PR DESCRIPTION
**Pull request by Patil2099**
Closes #3475
Removes all /en links except en-US

### CODE REVIEW

**Comment by Patil2099** [comment]
@Pomax  Can you Check this?

File: network-api/networkapi/buyersguide/templates/buyersguide_home.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h1 class="h1-heading text-center mt-4">Be Smart. Shop Safe.</h1>
           {% endif %}
         {% endif %}
 
-        <a class="product-image text-center mt-4 h-100 d-flex flex-column justify-content-between" href="/en/privacynotincluded/products/{{ product.slug }}">
+        <a class="product-image text-center mt-4 h-100 d-flex flex-column justify-content-between" href="privacynotincluded/products/{{ product.slug }}">
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/buyersguide_home.html]: We should keep the leading `/` 

i.e., `href="/privacynotincluded/products/{{ product.slug }}"`

**Comment by Pomax** [comment]
After investigating the original underlying error, it turns out that rather than removing /en/ we need to replace each instance with the user's current locale, based on their Django sessions, which is a fairly complicated bit of work.

As we don't currently have the capacity to walk contributors through the process to make that happen, we'll have to close this PR and do this work in-house instead.

Thank you so much for being willing to help out, and apologies that this turned out far more complicated than initially thought =(

File: network-api/networkapi/utility/middleware.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def __call__(self, request):
                     redirect_url = '{protocol}://{hostname}{path}'.format(
                         protocol=protocol,
                         hostname=hostnames[0],
-                        path='/en/mozfest/'
+                        path='/mozfest/'
```
Comment (mmmavis) [inline network-api/networkapi/utility/middleware.py]: not 100% sure if we need to keep `/en` here

**Pull request closed without merging**